### PR TITLE
Fix swupd-add-pkg

### PR DESCRIPTION
**Pull request by tmarcu**
Signed-off-by: Tudor Marcu <tudor.marcu@intel.com>

### CODE REVIEW

**Comment by tmarcu** [comment]
https://github.com/clearlinux/mixer-tools/pull/231 corresponding PR in mixer-tools.

**Comment by matthewrsj** [comment]
Also need to remove from Makefile.am

**Comment by tmarcu** [comment]
@matthewrsj Fixed this to just include the fixes required for it for the time being.

File: swupd-add-pkg [hunk 1]
```diff
@@ -71,15 +71,40 @@ if [[ -n "$pkg" && -n "$bundle" ]]; then
...
+mixver=$(cat $MIX_DIR/mixversion)
+
 # Copy the rpms into the local database location
 sudo mixer add-rpms --config "$BUILDCONF"
 sudo mixer bundle add "$bundle" os-core --config "$BUILDCONF"
-sudo mixer build chroots --config "$BUILDCONF" --new-chroots
-sudo mixer build update --config "$BUILDCONF" --new-swupd --increment
+# Use the original (non-merged) MoM for creating updates between custom content
+if [ $PREVIOUS_VER -eq 0 ]; then
+	sudo mixer build chroots --config "$BUILDCONF" --new-chroots
```
Comment (matthewrsj) [inline swupd-add-pkg]: `mixer build bundles` or are we just going with the alias until docs are updated?

**Pull request merged**